bot.py

```diff
@@ -553,7 +536,7 @@ async def remove_summoner(ctx, *, message):
             await asyncio.sleep(1)
 
         # converting the message into list of summoners
-        summoner_to_remove_input = message.split(",")
```

Comment:
@HelloFive you should've caught this. Were you copying pasting from other functions? you should still think about it